language/functional_tests/tests/testsuite/validator_set/reconfiguration.mvir

```diff
@@ -83,7 +82,7 @@ main() {
     return;
 }
 
-// check: events: []
+// not: ValidatorSetChangeEvent
```

Comment:
Should we also check the balance of the validators to make sure that gas are indeed distributed across the network?

Reply:
Agreed; I think we should at least create versions of the deleted `test_txn_fee_distribution_does_div.mvir` and `test_txn_fee_distribution_no_div.mvir` files that test the same thing in this new setting.

Reply:
I've added those two deleted tests back, but not in this file.